Failing test: Security Solution Cypress.x-pack/solutions/security/test/security_solution_cypress/cypress/e2e/explore/cases/attach_alert_to_case·cy·ts - Alerts timeline Privileges: read only "before each" hook for "should not allow user with read only privileges to attach alerts to existing cases" "before each" hook for "should not allow user with read only privileges to attach alerts to existing cases"

A test failed on a tracked branch

```
AssertionError: Timed out retrying after 150000ms: Expected <span.euiLoadingSpinner.css-1fsqg4c-euiLoadingSpinner-l> not to exist in the DOM, but it was continuously found.

Because this error occurred during a `before each` hook we are skipping the remaining tests in the current suite: `Alerts timeline`
    at waitForAlerts (webpack:///./tasks/alerts.ts:459:28)
    at waitForAlertsToPopulate (webpack:///./tasks/create_new_rule.ts:918:16)
    at Context.eval (webpack:///./e2e/explore/cases/attach_alert_to_case.cy.ts:36:28)
```

First failure: [kibana-on-merge - main](https://buildkite.com/elastic/kibana-on-merge/builds/93547#019d7919-bf6f-40a8-ba5c-d31aba48fbaf)

<!-- kibanaCiData = {"failed-test":{"test.class":"Security Solution Cypress.x-pack/solutions/security/test/security_solution_cypress/cypress/e2e/explore/cases/attach_alert_to_case·cy·ts","test.name":"Alerts timeline Privileges: read only \"before each\" hook for \"should not allow user with read only privileges to attach alerts to existing cases\" \"before each\" hook for \"should not allow user with read only privileges to attach alerts to existing cases\"","test.failCount":4}} -->

---

_This issue is tracked by the [Threat Hunting Tests Stability epic](https://github.com/elastic/security-team/issues/17524). See the epic description for instructions on how to work on this._

--- a/x-pack/solutions/security/test/security_solution_cypress/cypress/e2e/explore/cases/attach_alert_to_case.cy.ts
+++ b/x-pack/solutions/security/test/security_solution_cypress/cypress/e2e/explore/cases/attach_alert_to_case.cy.ts
@@ -17,7 +17,6 @@ import { visit } from '../../../tasks/navigation';
 
 import { ALERTS_URL } from '../../../urls/navigation';
 import { ATTACH_ALERT_TO_CASE_BUTTON, TIMELINE_CONTEXT_MENU_BTN } from '../../../screens/alerts';
-import { LOADING_INDICATOR } from '../../../screens/security_header';
 import { deleteAlertsAndRules } from '../../../tasks/api_calls/common';
 
 const loadDetectionsPage = (role: SecurityRoleName) => {
@@ -55,7 +54,6 @@ describe('Alerts timeline', () => {
   context('Privileges: can crud', { tags: ['@ess', '@serverless'] }, () => {
     beforeEach(() => {
       loadDetectionsPage(ROLES.platform_engineer);
-      cy.get(LOADING_INDICATOR).should('not.exist');
     });
 
     it('should allow a user with crud privileges to attach alerts to cases', () => {

